packages/core/src/elements/Table/FeTable.tsx

```diff
@@ -275,15 +275,22 @@ export const FeTable: FC<TableProps> = <T extends object>(props: TableProps<T>)
       <div ref={tableRef} className={classNames(prefixCls, props.className)} {...getTableProps()}>
         {props.toolbar && <FeTableToolbar />}
 
-        <FeTableTHead {...tableHeadProps} />
-        <FeTableTBody
-          loading={props.loading}
-          prefixCls={prefixCls}
-          prepareRow={prepareRow}
-          getTableBodyProps={getTableBodyProps}
-          renderExpandedComponent={props.renderExpandedComponent}
-          rows={tableRows}
-        />
+        <div
+          className={classNames(
+            `${prefixCls}__table-container`,
+            props.loading && `${prefixCls}__table-container-loading`
+          )}
+        >
+          <FeTableTBody
+            loading={props.loading}
+            prefixCls={prefixCls}
+            prepareRow={prepareRow}
+            getTableBodyProps={getTableBodyProps}
+            renderExpandedComponent={props.renderExpandedComponent}
+            rows={tableRows}
+          />
+          <FeTableTHead {...tableHeadProps} />
```

Comment:
why the FeTableThead is under the FeTableTBody? is this for z-index and order?

Reply:
I use the order of the flex blocks.
It allows avoiding the use of z-index with dom tree because z-index could borned a lot of problems.

Reply:
It is [here](https://github.com/frontegg/frontegg-react/pull/266/files#diff-39734fa7da22e890d7f1bc4933504f8820f30b9857e3fad9cd8d501856e26592R132)